styles.css

```diff
@@ -34,3 +34,56 @@
   cursor: pointer;
   text-decoration: underline;
 }
+
+.container {
+  margin-top: 40px;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+#inputText {
+  width: 600px;
+  height: 500px;
+}
```

Comment:
Please use classes and in this case use the rows and cols properties of the text-area
https://www.w3schools.com/tags/tag_textarea.asp